src/crimson/os/seastore/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(crimson_seastore_srcs
   extent_placement_manager.cc
   object_data_handler.cc
   seastore.cc
-  random_block_manager/nvme_manager.cc
+  random_block_manager/block_nvme_manager.cc
```

Comment:
block_nvme_manager.cc -> block_rbm_manager.cc

also the commit title should be `rename NVMeManager to BlockRBMManager`